components/vc/front/include/HW/HW_adc_componentSpecific.h

```diff
@@ -28,7 +28,7 @@ typedef enum
     ADC_BANK1_CHANNEL_L_SHK_DISP,
     ADC_BANK1_CHANNEL_PU1,
     ADC_BANK1_CHANNEL_BR_POT,
-    ADC_BANK1_CHANNEL_SPARE1,
+    ADC_BANK1_CHANNEL_BRK_TEMP,

```

Comment:
feels like there should be an FL and FR channel rather than just a single channel?